src/braket/experimental/autoqasm/api.py

```diff
@@ -200,11 +212,12 @@ def _autograph_optional_features() -> tuple[converter.Feature]:
 
 def _convert_main(
     f: Callable,
+    *,
```

Comment:
This line can be removed (for now, until we revisit in the follow up about the more strict APIs here)

Reply:
I'd prefer to enforce using argument names over position here, but I'm okay doing that in a separate PR. I'll revert for now